_posts/2024-04-01-kubecon-2024.md

```diff
@@ -0,0 +1,174 @@
+
+If you're running Terraform or OpenTofu setups at scale, this is surely a project to check out.
+One of the few downsides is that Atlantis expects to receive webhooks from your SCM.
+This requires Atlantis to be exposed on the public internet if you're using a SaaS-based SCM.
+There are workarounds:
+- limit the IPs that can access the endpoint to those linked to your SCM
+- Put a reverse proxy in front of Atlantis and heavily filter the traffic
+- (our favorite) use natively integrated CI solutions like AWS CodePipeline and "fake" the webhook from there
+
+## Startup lessons from 25 years and 5 startups & Panel: Venture Captial and Open Source
+
+These were two talks in the StartupFest co-located event of Kelsey Hightower.
+The first talk discussed six useful lessons for doing a startup.
+Tim Enwall from Fermyon shared his experiences and gave startup owners key things to consider when managing their startups.
+One of the takeaways that was mentioned multiple times during other talks as well was: don't be afraid of giving away one or two percent more of shares when discussing Venture Capital.
+The key reason is that the percentages don't make a difference in case the startup fails, 1 or 2 percent of 0 is still 0.
+On the other hand, if the startup does very well, you're at the point where it does not significantly affect your outcome either because you'll be earning tons of money anyway.
+
+Another interesting lesson Tim shared was that VCs are a necessary evil.
+I'm not sure if this was planned or not, but the talk directly afterward was a panel with VCs.
+The VCs shared how to evaluate potential investments are provided suggestions on how a startup can make itself more interesting to a VC.
+The two takeaways here were: the best way to some product-market-fit is by having paying customers using your solution and secondly, show the VC data that backs any claims you're making w.r.t. customer adoption and market interest in your product.
+
+## [Kubernetes Maintainers Read Mean Comments - Tim Hockin (Google) & Davanum Srinivas (AWS)](https://www.youtube.com/watch?v=sNRnB4rGZBI){:target="_blank" rel="noopener noreferrer"}
+
+Being an open-source developer can be a very satisfying and rewarding job.
+Engaging with a large community, particularly with famous open-source software, often leads to heartwarming feedback from dedicated users.
+Open-source is more than just the open development of a product.
+It has a widespread community, people contribute by either reporting issues on the issue tracker, by submitting a pull request to improve the software, participating in discussions on Slack or public forums, attending public meetings, ... .
+Sadly, it can also be a thankless job and not all people realize that open source software is something that anyone can contribute to, and does not really have any limitations as to how someone can contribute to.
```

Comment:
```suggestion
Sadly, it can also be a thankless job and not all people realize that open source software is something that anyone can contribute to, and it doesn't really have any limitations as to how someone can contribute either.
```